src/view/components/semantic/modules/dropdown/dropdown.ts

```diff
@@ -240,8 +241,13 @@ export class Dropdown implements AfterViewInit, OnDestroy, ControlValueAccessor
    */
   private _applyArrowNavFix($dropdown) {
     let $searchField = $dropdown.children('input.search')
+    let enterEvent = this.enter;
     $searchField.on('keyup', (event:any)=> {
       if (DO_NOT_SEARCH_ON_THESE_KEY_EVENTS[event.keyCode]) {
+        if (event.keyCode == 13 && enterEvent){
```

Comment:
I see your concern. So, 'enter' is an output, not an input. When you declare and set it to a value it will always stay that value. Consumers of that output are binding to it, not 'setting' the value, as is the case with @Input fields.